config/initializers/devise_token_auth.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+DeviseTokenAuth.setup do |config|
+  # By default the authorization headers will change after each request. The
+  # client is responsible for keeping track of the changing tokens. Change
+  # this to false to prevent the Authorization header from changing after
+  # each request.
+  # config.change_headers_on_each_request = true
+
+  # By default, users will need to re-authenticate after 2 weeks. This setting
+  # determines how long tokens will remain valid after they are issued.
+  # config.token_lifespan = 2.weeks
```

Comment:
```suggestion
  config.token_lifespan = 1.hour
```

さっきちょっとだけ話したのですが、 Vue.js の SPA で作っているので、ちょっとトークンの扱いが雑になってしまい、セキュリティ的に心配な面があります。
授業の課題なので、あまり気にしなくても良いことではあるのですが、せめてトークンの寿命を短くしておきたいなと思いました。

コメントアウト外して、１時間に設定していただけますか？ 🙏